actioncable/app/javascript/action_cable/connection_monitor.js

```diff
@@ -75,26 +73,31 @@ class ConnectionMonitor {
   }
 
   getPollInterval() {
-    const {min, max, multiplier} = this.constructor.pollInterval
-    const interval = multiplier * Math.log(this.reconnectAttempts + 1)
-    return Math.round(clamp(interval, min, max) * 1000)
+    const { staleThreshold, reconnectionBackoffRate } = this.constructor
+    const backoff = Math.pow(1 + reconnectionBackoffRate, Math.min(this.reconnectAttempts, 10))
+    const jitter = (this.reconnectAttempts > 0 ? reconnectionBackoffRate : 1) * Math.random()
+    return staleThreshold * 1000 * backoff * (1 + jitter)
```

Comment:
Love the `reconnectionBackoffRate` variable name and how readable the return statement is now!